vendor/diff-parsers-ext-parsewyc.patch

```diff
@@ -0,0 +1,978 @@
+--- ocaml-4.13-extended/lexer.mll
++++ parse-wyc/lib/lexer.mll
+@@ -253,14 +253,14 @@
+ 
+ let escaped_newlines = ref false
+ 
+ (* Warn about Latin-1 characters used in idents *)
+ 
+-let warn_latin1 lexbuf =
+-  Location.deprecated
++let warn_latin1 lexbuf = ignore lexbuf
++  (*Location.deprecated
+     (Location.curr lexbuf)
+-    "ISO-Latin1 characters in identifiers"
++    "ISO-Latin1 characters in identifiers"*)
+ 
+ let handle_docstrings = ref true
+ let comment_list = ref []
+ 
+ let add_comment com =
+@@ -272,63 +272,10 @@
+   in
+     add_comment com
+ 
+ let comments () = List.rev !comment_list
+ 
+-(* Error report *)
```

Comment:
This diff could be made smaller by commenting out the code instead of deleting it

Reply:
done